### PR TITLE
Moved to digital-link v1.2.4 & Updated other dependencies

### PR DESCRIPTION
**Pull request by clementh59**
The verifier and generator now verify if the check digit of the identifier is valid.

### CODE REVIEW

**Comment by clementh59** [comment]
FYI, the deploy preview failed because digital-link.js v1.2.4 isn't released yet.

File: package.json [hunk 1]
```diff
@@ -4,22 +4,23 @@
...
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.9.1",
-    "parcel-bundler": "^1.12.3",
-    "qrious": "^4.0.2",
-    "request": "^2.87.0"
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.28.0",
+    "parcel-bundler": "^1.12.5",
+    "request": "^2.88.2"
```
Comment (C-D-Lewis) [inline package.json:24]: `request` is deprecated, suggest using `fetch` if possible

File: src/generator.html [hunk 2]
```diff
@@ -186,7 +186,7 @@ <h2>Create Your GS1 Digital Link</h2>
     </div>
 
     <div class="fg-row fg-center mt-5 footer">
-      <a class="subtitle-xsmall" href="https://evrythng.com/gs1digitallink/" target="_blank">© EVRYTHNG 2020</a>
+      <a class="subtitle-xsmall" href="https://evrythng.com/gs1digitallink/" target="_blank">© EVRYTHNG 2022</a>
```
Comment (C-D-Lewis) [inline src/generator.html:189]: @domguinard Should this just be the new year, or a range, such as "2018 - 2022"?

Comment (clementh59) [reply src/generator.html:189]: I talked with him about it - only "2022" is fine

**Review by C-D-Lewis — APPROVED** [review]

**Pull request merged**